dependency-injection/2.0/_index.md

```diff
@@ -1,21 +1,24 @@
 ---
-title: "Jakarta Dependency Injection 2.0 (milestone)"
-date: 2020-05-18
+title: "Jakarta Dependency Injection 2.0"
+date: 2020-06-7
```

Comment:
@starksm64, This date has to be modified to meet the format requirements of yyyy-mm-dd.  @m0mus is correct that this needs to be 2020-06-07, or anything up to the current date of 2020-08-04.  This needs to be resolved before we can initiate a ballot otherwise the [Spec page has the DI releases](https://deploy-preview-221--jakartaee-specifications.netlify.app/specifications/dependency-injection/) in the wrong order.  Thanks.